### PR TITLE
Issue 4 view saved

### PR DESCRIPTION
**Pull request by hsu-isaac**
ajax-issue-4

### CODE REVIEW

**Comment by 16hsuisaac** [comment]
image

**Comment by hsu-isaac** [comment]
image

File: js/main.js [hunk 1]
```diff
@@ -76,10 +88,113 @@ function bookmark(event) {
...
+    } else {
+      p.textContent = 'Unknown';
+    }
+    p.setAttribute('class', 'inline padding-none text-align-right-mobile margin-bottom-none margin-top-none');
+    li.appendChild(div);
+    div.appendChild(p);
+    if (p2) {
+      div.appendChild(p2);
+    }
+    $ul.appendChild(li);
```
Comment (codingcodymiller) [inline js/main.js]: You should make this into its own function to generate a DOM tree for one dog, and then `return` that DOM tree so you can append it where the function is called.

Comment (codingcodymiller) [reply js/main.js]: Wait, isn't `dogListView` doing this work already?

File: js/main.js [hunk 2]
```diff
@@ -76,10 +88,113 @@ function bookmark(event) {
...
+}
 
+function showSaved(event) {
+  $homePage.setAttribute('class', 'home-page hidden');
+  $viewPage.setAttribute('class', 'view-page');
+  $savedButtonHeader.setAttribute('src', 'images/bookmark-pink.png');
+  $homeButtonHeader.setAttribute('src', 'images/house-black.png');
+  $savedButton.setAttribute('src', 'images/bookmark-pink.png');
+  $homeButton.setAttribute('src', 'images/house-black.png');
 }
```
Comment (codingcodymiller) [inline js/main.js]: You should make a `switchView` function that can receive an argument of the view to show, just like you did in `code-journal`. That way you don't need to create a new function every time you add a new view.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**